pytorch_lightning/trainer/training_loop.py

```diff
@@ -527,7 +530,7 @@ def run_training_epoch(self):
             # -----------------------------------------
             self.save_loggers_on_train_batch_end()
 
-            # update LR schedulers
+            # update plateau LR scheduler after metrics are logged
             self.update_lr_schedulers('step')
```

Comment:
Probably should add one more test (or update the existing ones) so both scheduler types are available to test for any duplicate `step` calls.

Reply:
say you have two schedulers, a regular one and a plateau. 

```
# here it updates the regular one, and skips the plateau
self.update_lr_schedulers('step', skip_plateau_scheduler=True)
...
# here how do you know you already stepped the first scheduler?
self.update_lr_schedulers('step')
```

Reply:
Because the current implementation is using a XOR for `skip_plateau` and `is_plateau`.

So the first call sets `skip_plateau=True` so only when `not is_plateau` will get updated

And the second call has `skip_plateau=False` so only when `is_plateau` gets updated

Reply:
Maybe skip_plateau_scheduler is not the best naming. It seems we are not skipping (updating everything) when skip is False. I'll rename is update_plateau_schedulers.